tracing/src/main/java/com/palantir/tracing/DeferredTracer.java

```
@@ -58,7 +58,7 @@
     private static final String DEFAULT_OPERATION = "DeferredTracer(unnamed operation)";
 
     @Nullable
-    private final String traceId;
+    private final CommonTraceState commonTraceState;
```

Let's increment `serialVersionUID` to `2L`. We don't need serialization backcompat here since it's primarily used in spark where we're guaranteed that libraries are always in sync, but the increment will help us fail loudly if cross-version ser/de is attempted.